src/app/data-analytics-dashboard/oncology/oncology-reports/oncology-summary-indicators/oncology-summary-indicators.component.ts

```diff
@@ -63,12 +68,25 @@ export class OncologySummaryIndicatorsComponent implements OnInit, AfterViewInit
     }, (error) => {
       console.error('Error', error);
     });
+    this.checkIsPdfReportAvailable();
+
   }
 
   public ngAfterViewInit() {
     this.cd.detectChanges();
   }
 
+  public checkIsPdfReportAvailable() {
+    const reportTypes: Array<any> = [];
+    const reports: Array<any> = OncologyReportConfig.reports;
+    reports.forEach(element => {
```

Comment:
I would really appreciate if you chose another name for **element** that is contextual to your function. Looking it up in your commit and contextualize it